doc/RowBasedReplication.md

```diff
@@ -116,6 +120,24 @@ We have future plans to:
 * Maintain a history of the schema changes that happen on all shards, so events
   can be parsed correctly in all cases.
 
+## Unsupported Features
+
+This part describes the features that are not supported for RBR in Vitess:
+
+* *Fractional timestamps for MariaDB*: not supported. This affects the objects
+  of type `TIMESTAMP`, `TIME` and `DATETIME`. The way that feature is
+  implemented in MariaDB, the binary logs do not contain enough information to
+  be parsed, but instead MariaDB relies on the schema knowledge. This is very
+  fragile. MySQL 5.6+ added new data types, and these are supported.
+
+* *JSON type in MySQL 5.7+*: the representation of these in the binlogs is a
```

Comment:
@alainjobart thanks for the clarification. We also use JSON columns very widely. But we're unlikely to be doing any sharding in the near future. So while I think this will be a blocker for us at some point, it probably doesn't have to be now. It would be nice to update the docs to say that we can support JSON columns with some work, though.

I currently use the Update Stream to refresh schemas on DDLs. I'm not yet doing horizontal or vertical sharding, so otherwise am not using the binlog.